.travis.yml

```diff
@@ -1,4 +1,4 @@
 cache: yarn
 language: node_js
-node_js: 16
+node_js: 14
```

Comment:
is this consistent across our repos?

Reply:
Looks like `container_core` (C3) and `integration_core` are using `14.16.1`. I'm happy to look at `.travis.yml` added to this repo too, even if we're removing this change. Though I'm seeing a unit test failure in the focus manager. Likely due to some JSDOM bug in Travis CI 🤔